src/components/Card.js

```diff
@@ -33,7 +65,13 @@ export class Card {
       .querySelector(".card__like-button")
       .classList.toggle("card__like-button_active");
   }
-  _handleDeletteButton() {
+
+  _openDeletePopup() {
+    deletePopup.open();
+  }
+
+  _handleDeleteButton() {
```

Comment:
Needs correction: please rename this method to `deleteCard` as it's not and event handler anymore (you should call it in index.js after the successful card deletion)